### ISSUE TITLE
stem_model m_a

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/MWinflect#4]
### masculine nouns ending in 'a'

We derive this list from the lexnorm-all2 list by the simple filter 
a) key1 ends in short vowel 'a'
b) lexnorm is precisely 'm'.

This excludes many adjectives and other nominals ending in 'a', since these will have more complex normalized lexnorm values, such as 'm:f:n', 'm:f#ikA':n'.

There are 49344 of these simple masculine nouns in 'a'. Their information is put into file:
 inputs/nominals/m_a.txt. For example, the two inputs from lexnorm-all2 are merged into one
input in m_a.txt:
```
579	akzara	a-kzara	m
592.1	akzara	a-kzara	m

becomes:
m_a	a-kzara	579,akzara:592.1,akzara

```

**Comment by funderburkjim** [comment]
### decline_file program

The decline_file program generates declensions based upon the model and stem  (first two fields) of
records in m_a.txt  (or one of the other files of inputs/nominals/ directory).
The output is written to a file in outputs/nominals/ directory under the same file name; e.g., in this
case to outputs/nominals/m_a.txt.

The format of the output files generated by decline_file is a sequence of lines, each with 3 tab-delimited
fields:
* model  (copied from input file)
* stem    (copied from input file - same as key2 in mw.txt)
* inflection  The declension table for this model and stem
### format of declension table
The declension table is represented as a string with 24
   parts (separated by colon), representing the singular, dual, plural of 8 cases. Symbolically
   `1s:1d:1p:2s:2d:2p:3s:3d:3p:4s:4d:4p:5s:5d:5p:6s:6d:6p:7s:7d:7p:8s:8d:8p`.  The common English
   names for the 8 cases are 1 = Nominative, 2 = Accusative, 3 = Instrumental, 4 = Dative,
   5 = Ablative, 6 = Genitive, 7 = Locative, 8 = Vocative.
   * missing values are represented by empty strings (such as vocative for personal pronouns)
   * Sometimes, one or more of the 24 declension cells will have alternate values; these are
      represented in csv form with a forward-slash ('/') as the separator.

**Comment by funderburkjim** [comment]
### decline_checks.txt

This file shows declension tables checked against various sources, such as the two shown above.
As it progresses, this can be used as a reference when algorithmic differences are introduced.

If others find the need, I could develop a web application to show these declensions with choice of
the user's model and key2.   I'll probably do this eventually, once the algorithms are stable.   One
feature would be to allow the user's choice of how to represent Sanskrit.  Since the internals of
the algorithms use the SLP1 spelling of Sanskrit words, it is easiest to show outputs, such as the tables
above, also in SLP1.

**Comment by gasyoun** [comment]
> easiest to show outputs, such as the tables
above, also in SLP1.

If our readers are bots - it will work best.

**Comment by drdhaval2785** [comment]
https://github.com/sanskrit-coders/indic_transliteration python package seems to support transliteration to and from SLP1 very well. So let us use it, if possible. So that custom transliteration code can be kept to bare minimum.

**Comment by gasyoun** [comment]
> So that custom transliteration code can be kept to bare minimum.

Exactly. Not even @SergeA can read it well, what to speak of other humans...

**Comment by funderburkjim** [comment]
Are you suggesting that all the inputs/outputs that I'm creating should be duplicated, so that there are
not only slp1 spellings but also IAST spellings?

**Comment by funderburkjim** [comment]
re the indic_transliteration package.

Does this package support accents?

**Comment by gasyoun** [comment]
> Internals can remain SLP1

Right.
